.github/2.0/cookbooks/Document.md

```diff
@@ -676,3 +676,48 @@ np.stack(da.get_attributes('embedding'))
  [4 5 6]
  [7 8 9]]
 ```
+
+### Access nested attributes from tags
+
+`Document` contains the `tags` field that can hold a map-like structure that can map arbitrary values.
+
+```python
+from jina import Document
+
+doc = Document(tags={'dimensions': {'height': 5.0, 'weight': 10.0}})
+
+doc.tags['dimensions']
+```
+
+```text
+{'weight': 10.0, 'height': 5.0}
+```
+
+In order to provide easy access to nested fields, the `Document` allows to access attributes by composing the attribute 
+qualified name with interlaced `__` symbols:
+
+```python
+from jina import Document
+
+doc = Document(tags={'dimensions': {'height': 5.0, 'weight': 10.0}})
+
+doc.tags__dimensions__weight
```

Comment:
I suggest educating the users to use the same way for retrieving tags information

```suggestion
doc.get_attributes('tags__dimensions__weight', 'tags__dimensions__height')
```

Reply:
I am showing in the access in Bulks later